apps/src/aichat/aichatApi.ts

```diff
@@ -167,6 +167,9 @@ export async function postAichatCompletionMessage(
     aichatContext,
   };
 
+  maxPollingTimeMs =
```

Comment:
Should this be * 1.5?

Reply:
🤦 the ms suffix should have been a hint, huh? Was remembering the constant from your first iteration, which was in seconds. Sorry!